src/load.ts

```diff
@@ -1,23 +1,9 @@
-import type FpJs from '@fingerprintjs/fingerprintjs-pro'
-import type { LoadOptions, GetResult } from '@fingerprintjs/fingerprintjs-pro'
+import FingerprintJS from '@fingerprintjs/fingerprintjs-pro'
+import type { LoadOptions, GetOptions, GetResult } from '@fingerprintjs/fingerprintjs-pro'
 import { pick } from './pick'
 
-declare global {
-  interface Window {
-    FingerprintJS: typeof FpJs
-  }
-}
-
-export const load = (loadOptions: LoadOptions, callback: (result: GetResult) => void) => {
-  const cdnUrl = `https://fpcdn.io/v3/${loadOptions.apiKey}/esm.min.js`
-  const fpPromise = new Promise((resolve, reject) => {
-    const script = document.createElement('script')
-    script.onload = resolve
-    script.onerror = reject
-    script.async = true
-    script.src = cdnUrl
-    document.head.appendChild(script)
-  }).then(() => window.FingerprintJS.load(pick(loadOptions, ['region']) as LoadOptions))
+export const load = (options: LoadOptions & GetOptions<boolean>, callback: (result: GetResult) => void) => {
+  const fpPromise = FingerprintJS.load(pick(options, ['region', 'endpoint']) as LoadOptions)
```

Comment:
For the same reason I recommend omitting fields instead of picking.

Also please add an `integrationInfo` parameter.